### ISSUE TITLE
Add to Opera Sidebar

### ISSUE THREAD

**Issue by bingling00** [vsmejkal/chrome-workspaces#22]
The sidebar function is better to use, and vivaldi can achieve it. Hope to add to the sidebar of opera

![image](https://user-images.githubusercontent.com/61322616/192295390-9565e38e-9eb0-4cb0-97f4-d114456ab132.png)

Thanks!This plug-in is really great

**Comment by imageslr** [comment]
How does the vivaldi achieve this? as your posted image above.

**Comment by bingling00** [comment]
> 

Copy the plug-in pop-up page address and add it to the sidebar.
 mine is： `extension://hpljjefgmnkloakbfckghmlapghabgfa/src/popup/popup.html`

**Comment by bingling00** [comment]
> How does the vivaldi achieve this? as your posted image above.
> 
> 维瓦尔第是如何做到这一点的？如您张贴的图像以上。

Copy the plug-in pop-up page address and add it to the sidebar.
mine is：‘ extension://hpljjefgmnkloakbfckghmlapghabgfa/src/popup/popup.html’

**Comment by imageslr** [comment]
That's awesome! min is `chrome-extension://hpljjefgmnkloakbfckghmlapghabgfa/src/popup/popup.html` in Chrome.

**Comment by bingling00** [comment]
> > mine is `chrome-extension://hpljjefgmnkloakbfckghmlapghabgfa/src/popup/popup.html` in Chrome.
> 
> do you mean that it's possible to add workspaces extension to the chrome sidebar?

Vivaldi can add any webpage to the sidebar, including workspaces extension webpages

**Comment by lazymonkey2** [comment]
> Vivaldi can add any webpage to the sidebar, including workspaces extension webpages

yes, but I had read "in Chrome" and was wondering if it was possibile to add it to the sidebar in Chrome too (guess it's not possible).
